xbmc/epg/EpgContainer.cpp

```diff
@@ -379,10 +374,11 @@ void CEpgContainer::InsertFromDatabase(int iEpgID, const std::string &strName, c
   else
   {
     // create a new epg table
-    epg = new CEpg(iEpgID, strName, strScraperName, true);
+    epg = new CEpg(-1);
```

Comment:
epgdb is a local variable in EpgDatabase::GetAll(), I dont call new() there because I don't know if its really inserted or only updated.